miniapp/CMakeLists.txt

```diff
@@ -9,34 +9,29 @@
 #
 
 add_library(DLAF_miniapp INTERFACE)
-target_link_libraries(DLAF_miniapp INTERFACE DLAF pika::pika)
+target_link_libraries(DLAF_miniapp INTERFACE DLAF)
```

Comment:
My personal preference is to have all direct dependencies explicitly listed. Since the miniapp headers include pika headers directly I would leave `pika::pika` here. In practice it doesn't make a difference of course, and feel free to ignore this.